app/domain/operations/families/verifications/dmf_determination/request_dmf_determination.rb

```diff
@@ -0,0 +1,144 @@
+
+          # @return [ Success ] Job successfully completed
+          def call(payload)
+            valid_payload = yield validate(payload)
+            family = yield find_family(valid_payload)
+
+            @job = yield find_job(valid_payload[:job_id])
+            values = yield construct_request_values(family.hbx_assigned_id)
+
+            @transmission = yield build_and_create_request_transmission(values)
+            @transaction = yield build_and_create_request_transaction(values, family)
+
+            payload = yield build_cv3_family_payload(family)
+            event = yield build_event(payload)
+            result = yield publish(event)
+
+            Success(result)
+          end
+
+          private
+
+          def validate(payload)
+            return handle_dmf_error('ALIVE_STATUS is not enabled for this env') unless EnrollRegistry.feature_enabled?(:alive_status)
+            return handle_dmf_error('Missing param :family_hbx_id') unless payload[:family_hbx_id].present?
+            return handle_dmf_error('Missing param :job_id') unless payload[:job_id].present?
+
+            Success(payload)
+          end
+
+          def find_family(params)
+            family = Family.find_by(hbx_assigned_id: params[:family_hbx_id].to_i)
+            Success(family)
+          rescue Mongoid::Errors::DocumentNotFound
+            handle_dmf_error("Family with hbx_id #{params[:family_hbx_id]} not found")
+          end
+
+          def find_job(job_id)
+            job = ::Transmittable::Job.find_by(job_id: job_id.to_s)
+            Success(job)
+          rescue Mongoid::Errors::DocumentNotFound
+            handle_dmf_error("Could not find Transmittable::Job with job_id #{job_id}")
+          end
+
+          def construct_request_values(family_hbx_id)
+            Success({
+                      key: @job&.key,
+                      title: @job&.title,
+                      description: @job&.description,
+                      correlation_id: family_hbx_id.to_s,
+                      started_at: @job&.started_at,
+                      publish_on: @job&.publish_on
+                    })
+          end
+
+          def build_and_create_request_transmission(values)
+            values[:key] = :dmf_determination_request
+            values[:event] = 'initial'
+            values[:state_key] = :initial
+            values[:job] = @job
+
+            create_request_transmission(values, @job)
+          end
+
+          def build_and_create_request_transaction(values, family)
+            values[:key] = :dmf_determination_request
+            values[:event] = 'initial'
+            values[:state_key] = :initial
+            values[:transmission] = @transmission
+            values[:subject] = family
+
+            create_request_transaction(values, @job)
+          end
+
+          def build_cv3_family_payload(family)
+            cv3_family = Operations::Transformers::FamilyTo::Cv3Family.new.call(family)
```

Comment:
Can you use this validate the cv3_family.`AcaEntities::Operations::CreateFamily` before save this.